internal/jimm/cloud.go

```diff
@@ -224,14 +224,6 @@ func (j *JIMM) AddHostedCloud(ctx context.Context, user *openfga.User, tag names
 		return errors.E(op, err)
 	}
 
-	allowedAddModel, err := user.IsAllowedAddModel(ctx, region.Cloud.ResourceTag())
```

Comment:
Should've been a separate PR.

Reply:
yes, it should, but i (we) keep forgetting about it :)